lib/sql.js

```diff
@@ -1159,7 +1159,12 @@ SQLConnector.prototype._buildWhere = function(model, where) {
         }
       }
     }
-    whereStmts.push(stmt);
+    var columnRestricted = false;
+    whereStmts.forEach(function(stmt) {
+      if (columnRestricted) return;
+      columnRestricted = stmt.sql.indexOf(columnName) >= 0;
+    });
+    if (!columnRestricted) whereStmts.push(stmt);
```

Comment:
This change will be problematic as we can use the same column multiple times in the same statement, for example:

`WHERE age > 18 AND age < 30`.